srcpkgs/pdf2djvu/patches/fix-c17.patch

```diff
@@ -0,0 +1,11 @@
+--- a/autoconf.mk.in  2022-08-09 23:28:01.000000000 +0300
++++ b/autoconf.mk.in2 2022-08-12 01:16:49.770542275 +0300
+@@ -13,7 +13,7 @@
+ 
+ # Compiler, etc.:
+ CXX = @CXX@
+-CXXFLAGS = @OPENMP_CXXFLAGS@ @CXXFLAGS@ @DJVULIBRE_CFLAGS@ @POPPLER_CFLAGS@ @GRAPHICSMAGICK_CFLAGS@ @XMP_CFLAGS@
++CXXFLAGS = -std=c++17 @OPENMP_CXXFLAGS@ @CXXFLAGS@ @DJVULIBRE_CFLAGS@ @POPPLER_CFLAGS@ @GRAPHICSMAGICK_CFLAGS@ @XMP_CFLAGS@
```

Comment:
this can be done by adding `export CXXFLAGS="-std=c++17"` in the template

Reply:
Won't this override the rest of the CXXFLAGS ?

Reply:
no